src/common/logging/Logging.h

```diff
@@ -12,6 +12,8 @@
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
 #endif
 
+#define SESSIONS_TELEMETRY_MARKER -9999
```

Comment:
Is this the constant to use for telemetry? Not clear, maybe add a comment?

Reply:
It's not for all telemetry lines and is nothing for telemetry client to consume. Telemetry client has to pick up log lines marked with '[TELEMETRY]' as PR description says. This is for internal use, to mark trace lines for whole sessions. See it used in Asb.c and in TelemetryUtils.c.